src/main/java/seedu/address/model/Model.java

```diff
@@ -157,6 +157,7 @@
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredModuleList(Predicate<Module> predicate);
+
```

Comment:
Probably shouldn't add blank lines in other parts of the code else you might create merge conflicts 😳!

Reply:
Sorry this is because of the merging isn't it. My bad.